# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

SearchForDogActionGoal action_goal
SearchForDogActionResult action_result
SearchForDogActionFeedback action_feedback
